hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/handlers/ContractCallLocalHandlerTest.java

```diff
@@ -181,21 +181,6 @@ void validateFailsIfNoContractOrTokenTest() {
         assertThatThrownBy(() -> subject.validate(context)).isInstanceOf(PreCheckException.class);
     }
 
-    @Test
```

Comment:
Because `contract.deleted()` that is mocked is no longer called and I either have to make it lenient or delete the mock. And I feel both of them will make the test useless. But if you think it's better to make it a positive test I would

Reply:
Hmm. I do see what you're saying.  Might it be valuable, for a postive test, to assert (via mockito) that `contract.deleted()` is _never_ called in this test?  That would be valid, I think.